src/app/Upload.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
+  let files = new Map<String, File>();
+
+  const handleDrop = (event: DragEvent) => {
+    console.log("File(s) dropped in the upload zone");
+    // Prevent default behavior (Prevent file from being opened)
+    event.preventDefault();
+
+    if (event.dataTransfer == null) {
+      return;
+    } else if (event.dataTransfer.items) {
+      // Use DataTransferItemList interface to access the file(s)
+      for (let i = 0; i < event.dataTransfer.items.length; i++) {
+        // If dropped items aren't files, reject them
+        if (event.dataTransfer.items[i].kind === "file") {
+          const file = event.dataTransfer.items[i].getAsFile();
+          if (file !== null) {
+            files.set(file.name, file);
+          }
+        }
+      }
+    } else {
+      // Use DataTransfer interface to access the file(s)
+      for (let i = 0; i < event.dataTransfer.files.length; i++) {
+        const file = event.dataTransfer.files[i];
+        alert(file);
+        files.set(file.name, file);
+      }
+    }
+    alert("File(s) dropped: " + files.size);
+  };
+
+  const handleDragOver = (event: DragEvent) => {
+    console.log("File(s) entered the upload zone");
+    // Prevent default behavior (Prevent file from being opened)
+    event.preventDefault();
+  };
+</script>
+
+<style>
+  #upload-zone {
```

Comment:
Since Svelte uses scoped CSS, it's not necessary to use `id`s for style. If you create a class here, it is visible **only within `Upload.svelte`! https://svelte.dev/tutorial/styling

Please change this to a class:

```suggestion
  .upload-zone {
```